tooling/t.js

```diff
@@ -0,0 +1,45 @@
+const fs = require("fs");
+
+const LOCALE = getProjectMeta().LOCALE;
+
+function t(key, args = {}, forceLangToUse) {
+  const loc = getProjectMeta().LOCALE;
+  // Get key from ./locales/{locale}/comments.json
+  // Read file and parse JSON
+  const locale = forceLangToUse ?? loc === "undefined" ? "en" : loc;
+  const comments = require(`./locales/${locale}/comments.json`);
+
+  // Get value from JSON
+  const value = comments[key];
+  // Replace placeholders in value with args
+  const result =
+    Object.values(args)?.length > 0
+      ? value.replace(/\{\{(\w+)\}\}/g, (_, m) => args[m])
+      : value;
+  // Return value
+  return result;
+}
```

Comment:
This is a little weird.  It's not a huge deal, but the translation function's job should just be translation, rather than "figuring out the locale and then translating".  I'd refactor it slightly, so that there are two functions `t` and a helper that t calls to do the translation after it determines the locale.

Reply:
I mean, it is, as things go, a tiny function. So, I am not fussed about it's current format.

Reply:
Yeah, fair enough. I was being overly picky.